### ISSUE TITLE
tripl/tripl.py", line 586, in _entity_lookup - AttributeError: 'NoneType' object has no attribute 'keys'

### ISSUE THREAD

**Issue by ghost** [metasoarous/tripl#14]
The following code (mostly copy & paste from the readme):
```python
from tripl import tripl


def cft_cons(name):
    return tripl.entity_cons('cft.type:' + name, 'cft.' + name)


def main():
    subject = cft_cons('subject')

    # Next our schema

    schema = {
        'cft.seq:timepoint': {'db:valueType': 'db.type:ref',
                              'db:cardinality': 'db.cardinality:many'},
        'cft.seq:subject': {'db:valueType': 'db.type:ref'}}

    ts = tripl.TripleStore(schema=schema, default_cardinality='db.cardinality:one')

    ts.assert_facts([
        subject(id='QA255')],
        id_attrs=['cft.timepoint:id', 'cft.seq:id', 'cft.subject:id'])
```

Causes an exception:
```
Traceback (most recent call last):
  File ".../venv/bin/...", line 11, in <module>
    load_entry_point('...', 'console_scripts', '...')()
  File ".../.../__init__.py", line 38, in main
    id_attrs=['cft.timepoint:id', 'cft.seq:id', 'cft.subject:id'])
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 521, in assert_facts
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 499, in assert_fact
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 472, in _assert_dict
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 448, in _resolve_eid
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 448, in <dictcomp>
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 591, in match
  File "build/bdist.linux-x86_64/egg/tripl/tripl.py", line 586, in _entity_lookup
AttributeError: 'NoneType' object has no attribute 'keys'
```

Do you have a hint at what might be causing this?

I am using Python 2.7.16 and unmodified Tripl from master branch.

**Comment by metasoarous** [comment]
I'm reproducing this now; I had previously failed because I didn't notice you were using `id_attrs`. I think I wrote that as a design idea into the README, but never actually implemented it because we didn't use it. Now that this is clearer, we should be able to figure out what is going on here.

**Comment by metasoarous** [comment]
This is now fixed in aa99ecf.

**Comment by ghost** [comment]
Thanks!
